shopinvader_api_payment/routers/payment.py

```diff
@@ -0,0 +1,184 @@
+# Copyright Odoo SA (https://odoo.com)
+# Copyright 2024 ACSONE SA (https://acsone.eu).
+# @author Stéphane Bidoul <stephane.bidoul@acsone.eu>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+from typing import Annotated
+from urllib.parse import urljoin
+
+from fastapi import APIRouter, Depends, HTTPException, Request
+
+from odoo import api, models
+
+from odoo.addons.fastapi.dependencies import odoo_env
+from odoo.addons.payment.models.payment_provider import PaymentProvider
+
+from ..schemas import (
+    PaymentDataWithMethods,
+    PaymentProvider as PaymentProviderSchema,
+    TransactionCreate,
+    TransactionProcessingValues,
+)
+from .utils import Payable
+
+_logger = logging.getLogger(__name__)
+
+payment_router = APIRouter(tags=["payment"])
+
+
+@payment_router.get("/payment/methods")
+def pay(
+    payable: str,
+    odoo_env: Annotated[api.Environment, Depends(odoo_env)],
+):
```

Comment:
```suggestion
) -> PaymentDataWithMethods:
```

Or reponse_model? We need to declare them on all routes.